src/components/modules/toolbar/blockSettings.ts

```diff
@@ -60,7 +60,7 @@ export default class BlockSettings extends Module {
   /**
    * Block settings UI HTML elements
    */
-  public nodes: {[key: string]: HTMLElement} = {
+  public nodes = {
```

Comment:
why not use the new `ModuleNodes` type in generic instead of declaring nodes here?